sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Release History
+## 5.1.1 (2021-08-11)
+### Key Bug Fixed
```

Comment:
The title of this section should be "Bugs Fixed". 
cc: @g2vinay to confirm.

Reply:
https://azure.github.io/azure-sdk/policies_releases.html#change-logs

Yes.  Will update it to `Bugs Fixed`